pkg/workloads/cortex/tf_api/api.py

```diff
@@ -445,18 +445,15 @@ def get_signature(app_name, api_name):
 def download_dir_external(ctx, s3_path, local_path):
     util.mkdir_p(local_path)
     bucket_name, prefix = ctx.storage.deconstruct_s3_path(s3_path)
-    objects = ctx.storage.list_objects(s3_path)["Contents"]
+    objects = ctx.storage.list_objects(s3_path)
     version = prefix.split("/")[-1]
     for obj in objects:
-        local_key = obj["Key"].lstrip(prefix[: -len(version)])
+        local_key = obj[len(prefix) - len(version) :]
```

Comment:
I think `list_objects()` should return paths relative to the dir that was passed in. Can you update the implementation to do that (and confirm the local implementation does the same)?